attributes/default.rb

```diff
@@ -3,8 +3,8 @@
 include_attribute "ndb"
 include_attribute "kzookeeper"
 
-default['hops']['versions']                    = "2.8.2.2,2.8.2.3,2.8.2.4,2.8.2.5,2.8.2.6,2.8.2.7,2.8.2.8,2.8.2.9,2.8.2.10"
-default['hops']['version']                     = "3.2.0.0-RC3"
+default['hops']['versions']                    = "2.8.2.2,2.8.2.3,2.8.2.4,2.8.2.5,2.8.2.6,2.8.2.7,2.8.2.8,2.8.2.9,2.8.2.10,3.2.0.0"
```

Comment:
@tkakantousis We do not have any changes in RC releases. So this should be fine. Tagging Gautier @berthoug  here just in case I am wrong.

Reply:
The version stored in here is the version for (in the name) of the sql files for hops. These files do not include the minor version (RC) because we do not update the database in minor version. So it should be 3.2.0.0